src/dispatch/static/dispatch/src/project/NewEditSheet.vue

```diff
@@ -93,7 +93,7 @@
                     type="number"
                     min="1"
                     pattern="\d+"
-                    placeholder="2080"
+                    placeholder="1640"
```

Comment:
```suggestion
                    placeholder="2080"
```

Reply:
We should also update the hint `hint="Number of hours to average employee host."`. We could say something along the lines of "Number of working hours in a year to calculate hourly rate."

Reply:
Ah, thank you.